glfw/src/Window.zig

```diff
@@ -48,16 +48,16 @@ pub const InternalUserPointer = struct {
     user_pointer: ?*c_void,
 
     // Callbacks to be invoked by wrapper functions.
-    setPosCallback: ?fn (window: Window, xpos: isize, ypos: isize) void,
-    setSizeCallback: ?fn (window: Window, width: isize, height: isize) void,
+    setPosCallback: ?fn (window: Window, xpos: i32, ypos: i32) void,
+    setSizeCallback: ?fn (window: Window, width: i32, height: i32) void,
     setCloseCallback: ?fn (window: Window) void,
     setRefreshCallback: ?fn (window: Window) void,
     setFocusCallback: ?fn (window: Window, focused: bool) void,
     setIconifyCallback: ?fn (window: Window, iconified: bool) void,
     setMaximizeCallback: ?fn (window: Window, maximized: bool) void,
-    setFramebufferSizeCallback: ?fn (window: Window, width: isize, height: isize) void,
+    setFramebufferSizeCallback: ?fn (window: Window, width: i32, height: i32) void,
```

Comment:
Ah, additionally, we should change `width` and `height` here to a `u32` not `i32`, since we determined in https://github.com/hexops/mach/issues/114#issuecomment-987910915 that it should be unsigned.